src/index.html

```diff
@@ -8,7 +8,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
This needs to be put back in as it provides Font-Awesome for the menu icons currently but we will be using icons more in the future.

Reply:
That is an error by me!  Had a few google fonts in there and must have accidentally removed it!